contracts/utils/ExpiringMetaTxForwarder.sol

```diff
@@ -51,7 +50,35 @@ contract ExpiringMetaTxForwarder is EIP712, IExpiringMetaTxForwarder {
         ExpiringMetaTx calldata metaTx,
         bytes calldata signature
     ) external override returns (bytes memory returndata) {
-        bytes32 metaTxHash = _hashTypedDataV4(
+        bytes32 metaTxHash = processMetaTx(metaTx);
+        require(
+            metaTxHash.recover(signature) == metaTx.from,
+            "Invalid signature"
+        );
+        emit ExecutedMetaTx(metaTxHash);
+        returndata = metaTx.to.functionCall(
+            abi.encodePacked(metaTx.data, metaTx.from)
+        );
+    }
+
+    /// @notice Called by a meta-tx source to prevent it from being executed
+    /// @dev This can be used to nullify meta-txes that were issued
+    /// accidentally, e.g., with an unreasonably large expiration timestamp,
+    /// which may create a dangling liability
+    /// @param metaTx Meta-tx
+    function nullify(ExpiringMetaTx calldata metaTx) external override {
```

Comment:
agree with @acenolaza should be changed to something like `cancelMetaTx()`

Reply:
I will use `cancel()` similar to `execute()`. The argument is a part of the function signature so it's clear that it's a meta-tx that's being canceled.